File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        nums.sort()
        l = 0;
        r = len(nums)-1;

        while l < r:

             if nums[l]+nums[r] == target:
                return [l,r];
             elif nums[l]+nums[r] > target:
                r= r-1
             else:
                l=l+1
        return [];

File: Data Structures & Algorithms/jump-game/submission-0.java
class Solution {
    public boolean canJump(int[] nums) {
        int i=0;
        while(i<nums.length){
            
            if(nums[i]==0){
                return false;
            }
            i = i + nums[i];
            System.out.println(i);
            if(i>=nums.length-1){
                return true;
            }
        }

        return i>=nums.length;
    }
}
